Digitall.Warehouse.Application/Contracts/Requests/Products/CreateProductRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Digitall.Warehouse.Application.Contracts.Shared;
+
+namespace Digitall.Warehouse.Api.Contracts.Requests.Products;
+
+public class CreateProductRequest
+{
+    public string Title { get; set; } = default!;
```

Comment:
Do we need product without name? Maybe is good approach to add  [Required] on some properties

Reply:
No need it is already required and will no let the developer to pass element empty. As soon as you define a property as non nullable the model validation of the request takes care of it and will return an error back to the API consumer.
This is the piece of code where we shapre the validation error from .net framework into response we expect:
![image](https://github.com/tpilitis/Digitall.Warehouse/assets/19794485/33f2c0d6-1495-4ef7-aa92-1c78df766fb5)

Reply:
![Uploading image.png…]()